web-ui/src/pages/PulsePage.jsx

```diff
@@ -15,9 +15,9 @@ const PulsePage = () => {
   const csrf = selectCsrfToken(state);
 
   const [externalComment, setExternalComment] = useState('');
-  const [externalScore, setExternalScore] = useState(0);
```

Comment:
Updated default internal and external scores.
These are 0-based in the code (not in the database), so 2 is in the middle.